website/src/css/custom.css

```diff
@@ -0,0 +1,192 @@
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+:root {
+  --ifm-color-primary: #7919FF;
+  --ifm-color-primary-dark: #6a16e6;
+  --ifm-color-primary-darker: #5e13d4;
+  --ifm-color-primary-darkest: #4f10b8;
+  --ifm-color-primary-light: #8a1dff;
+  --ifm-color-primary-lighter: #9422ff;
+  --ifm-color-primary-lightest: #a02aff;
+  --ifm-code-font-size: 95%;
+  --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+  --ifm-code-background: #0A2E1B;
+}
+
+/* For readability concerns, you should choose a lighter palette in dark mode. */
+[data-theme='dark'] {
+  --ifm-color-primary: #9A52FF;
+  --ifm-color-primary-dark: #883ff5;
+  --ifm-color-primary-darker: #7732e2;
+  --ifm-color-primary-darkest: #6625cf;
+  --ifm-color-primary-light: #a565ff;
+  --ifm-color-primary-lighter: #b078ff;
+  --ifm-color-primary-lightest: #bb8bff;
+  --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
+}
+
+.button{
+  margin-left: 10px;
+}
+
+.hero--primary {
+  --ifm-hero-background-color: #232325;
+}
+
+.hero__title {
+  color: #FFFFFF;
+  font-size: 3.5rem;
+  font-family: 'Work Sans';
+}
+
+.hero__subtitle {
+  color: #FFFFFF;
+  display: inline-flex;
+  font-size: 1.125rem;
+  gap: 2rem;
+  font-family: 'Work Sans';
+  width: 54rem;
+}
+
+
+.button.button--secondary {
+  color: #FFFFFF;
+  background: #7919FF;
+  border: none;
+  font-family: 'Work Sans';
+  font-size: 1rem;
+  padding: 0.5rem 1rem 0.5rem 1rem;
+  font-weight: 700;
+}
+
+.button--lg {
+  --ifm-button-size-multiplier: 1;
+}
+
+
+.featureSvg_src-components-HomepageFeatures-styles-module{
+  height: 2.5rem!important;
+  width: 2.5rem!important;
+}
+
+code {
+  --ifm-code-background:#0A2E1B;
+  color: #5BEB9D;
+  font-family: 'Work Sans';
+}
+
+h3{
+  --ifm-h3-font-size: 1.3rem;
+  font-family: 'Work Sans';
+  font-weight: 700;
+}
+
+p {
+  font-size: 0.875rem;
+}
+
+.features_src-components-HomepageFeatures-styles-module{
+  padding: 10rem 0 !important;
+  background:#111112;
+}
+
+.footer--dark {
+  --ifm-footer-background-color: #232325;
+}
+
+[data-theme="light"] .features_src-components-HomepageFeatures-styles-module {
```

Comment:
You could use scss to group these better to clean it up. https://docusaurus.io/docs/styling-layout#sassscss